src/status_im/contexts/profile/settings/screens/notifications/view.cljs

```diff
@@ -0,0 +1,122 @@
+(ns status-im.contexts.profile.settings.screens.notifications.view
+  (:require
+    [quo.core :as quo]
+    [react-native.core :as rn]
+    [react-native.platform :as platform]
+    [status-im.common.events-helper :as events-helper]
+    [status-im.config :as config]
+    [status-im.feature-flags :as ff]
+    [utils.i18n :as i18n]
+    [utils.re-frame :as rf]))
+
+(defn toggle-notifications-enabled
+  [notifications-enabled?]
+  (rf/dispatch [:push-notifications/switch (not notifications-enabled?)]))
+
+(defn toggle-messenger-notifications
+  [messenger-notifications-enabled?]
+  (rf/dispatch [:notifications/messenger-notifications-switch (not messenger-notifications-enabled?)]))
+
+(defn toggle-news-notifications
+  [news-notifications-enabled?]
+  (rf/dispatch [:notifications/news-notifications-switch (not news-notifications-enabled?)]))
+
+(defn toggle-non-contact-notifications
```

Comment:
I see that these event weren't added in this PR, but as a general idea - we do not need to pass the value in `toggle` event, because we already have value in db and it can be toggled within the event itself.